components/NavBar/navbar.module.css

```diff
@@ -36,3 +41,11 @@
   height: 100%;
   border-radius: 50%;
 }
+
+.login {
+  align-self: center;
+  color: white;
```

Comment:
The Navbar width is more in mobile device. Also can add link to shop and cart page back in the nav bar as they were there. Also user photo will be there

Reply:
OK.
I'll check it and revert it.

Reply:
Do i have to check for only Desktop view or for mobile view also??